string term_id
time stamp
string src
memory/Term term
